modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -46,10 +48,21 @@
 
 #include "api/gbackend_priv.hpp" // FIXME: Make it part of Backend SDK!
 
+#include "executor/conc_queue.hpp"
+
 #if INF_ENGINE_RELEASE < 2021010000
 #include "ie_compound_blob.h"
 #endif
 
+#if defined(HAVE_TBB)
+#  include <tbb/concurrent_queue.h>  FIXME: drop it from here!
```

Comment:
why drop?

Reply:
It should in common place I believe, because the streaming executer uses it as well.

Reply:
`conc_queue` isn't used in there anymore.